src/Compilers/VisualBasic/Portable/CodeGen/EmitExpression.vb

```diff
@@ -2158,6 +2186,44 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeGen
             _builder.EmitOpCode(ILOpCode.Unbox_any)
             EmitSymbolToken(type, syntaxNode)
         End Sub
+
+        Private Sub EmitMethodDefIndexExpression(node As BoundMethodDefIndex)
+            Debug.Assert(node.Method.IsDefinition)
+            Debug.Assert(node.Type.SpecialType = SpecialType.System_Int32)
+            _builder.EmitOpCode(ILOpCode.Ldtoken)
+            EmitSymbolToken(node.Method, node.Syntax, encodeAsRawDefinitionToken:=True)
+        End Sub
+
+        Private Sub EmitMaximumMethodDefIndexExpression(node As BoundMaximumMethodDefIndex)
+            Debug.Assert(node.Type.SpecialType = SpecialType.System_Int32)
+            _builder.EmitOpCode(ILOpCode.Ldtoken)
+            _builder.EmitGreatestMethodToken()
+        End Sub
+
+        Private Sub EmitModuleVersionIdLoad(node As BoundModuleVersionId)
+            _builder.EmitOpCode(ILOpCode.Ldsfld)
+            _builder.EmitToken(_module.GetModuleVersionId(_module.Translate(node.Type, node.Syntax, _diagnostics), node.Syntax, _diagnostics), node.Syntax, _diagnostics)
+        End Sub
+
+        Private Sub EmitModuleVersionIdStore(node As BoundModuleVersionId)
+            _builder.EmitOpCode(ILOpCode.Stsfld)
+            _builder.EmitToken(_module.GetModuleVersionId(_module.Translate(node.Type, node.Syntax, _diagnostics), node.Syntax, _diagnostics), node.Syntax, _diagnostics)
```

Comment:
Consider extracting an `EmitModuleVersionIdToken()` helper method for use in `Load` and `Store`. Similar comment for `EmitInstrumentationPayloadRootToken()` below.